reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -76,16 +82,16 @@ receiving an error such as:
 Language server startup failed.
 ```
 
-To manually approve PowerShell Editor Services and the PowerShell extension for VSCode, open a
-PowerShell prompt and run the following command:
+To manually approve PowerShell Editor Services and the PowerShell extension for Visual Studio Code,
+open a PowerShell prompt and run the following command:
```

Comment:
Might be worth mentioning in this section that constrained language mode isn't currently supported

Reply:
added